modules/bvl_feedback/ajax/open_bvl_feedback_thread.php

```diff
@@ -13,8 +13,27 @@
 ini_set('default_charset', 'utf-8');
 require_once "bvl_panel_ajax.php";
 
-if (isset($_POST['feedbackID']) && isset($_POST['candID'])) {
-    $feedbackThread->openThread($_POST['feedbackID']);
+try {
+    $openedthreadcount = $feedbackThread->openThread($_POST['feedbackID']);
```

Comment:
It would be good to have an isset check for feedbackID, as you did in the other file